account_sequence_option/data/account_sequence_option.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
```

Comment:
You may want to add a noupdate here in order to avoid changing use_sequence_option to False upon update of the module